store/sqlstore/channel_store.go

```diff
@@ -1793,7 +1793,41 @@ func (s SqlChannelStore) GetMemberCountsByGroup(channelID string, includeTimezon
 	selectStr := "GroupMembers.GroupId, COUNT(ChannelMembers.UserId) AS ChannelMemberCount"
 
 	if includeTimezones {
-		selectStr = "GroupMembers.GroupId, COUNT(ChannelMembers.UserId) AS ChannelMemberCount, COUNT( DISTINCT Users.Timezone ) AS ChannelMemberTimezonesCount"
+		endOfFirstKey := `LOCATE(':', Users.Timezone) + 2`
+		lengthOfFirstValue := `LOCATE(',', Users.Timezone) - LOCATE(':', Users.Timezone) - 3`
+		endOfSecondKey := `LOCATE(',', Users.Timezone) + 19`
+		lengthOfSecondValue := `LOCATE('useAutomaticTimezone', Users.Timezone) - 22 - LOCATE(',', Users.Timezone)`
```

Comment:
this query is now tightly coupled with the order of the keys in this map... not sure if theres a way to avoid while still computing everything in a single query. On the bright side if someone modifies this column at all these tests will break, but then this query will need to be modified again.